tests/libkubevirt/config/netbinding.go

```diff
@@ -29,15 +29,13 @@ import (
 	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
 )
 
-func WithNetBindingPlugin(name string, netBindingPlugin v1.InterfaceBindingPlugin) error {
-	return RegisterKubevirtConfigChange(func(c v1.KubeVirtConfiguration) (*patch.PatchSet, error) {
-		return registerBindingPugins(c, name, netBindingPlugin)
-	})
+func WithNetBindingPlugin(name string, netBindingPlugin v1.InterfaceBindingPlugin) KvChangeOption {
+	return func(kv v1.KubeVirt) *patch.PatchSet {
+		return registerBindingPugins(kv.Spec.Configuration, name, netBindingPlugin)
+	}
 }
 
-func registerBindingPugins(config v1.KubeVirtConfiguration, name string, binding v1.InterfaceBindingPlugin) (
-	*patch.PatchSet, error,
-) {
+func registerBindingPugins(config v1.KubeVirtConfiguration, name string, binding v1.InterfaceBindingPlugin) *patch.PatchSet {
```

Comment:
There is a little typo here (existed prior to this PR), would you mind fixing it?
```suggestion
func registerBindingPlugins(config v1.KubeVirtConfiguration, name string, binding v1.InterfaceBindingPlugin) *patch.PatchSet {

```